performant-software/Annotation-Studio
# Ingested user, document, and most annotation data from COVE editions

### What does this PR do?

Brings COVE data into Studio.

### How to test

1. Go to http://cove-staging-pr-85.herokuapp.com/
1. Log in (as an admin user -- let me know which account you're using and I will grant you permissions)
1. Go to http://cove-staging-pr-85.herokuapp.com/documents/
1. Click the "All" tab
1. Note that there are 10 or so documents; those are from COVE editions
1. Click on of them to view it
1. Click the gear menu at top right to open the annotation filters
1. Click "All" on the top filter, to see annotations made by all users.
1. Note that there are many, made by different users.

### What's not covered by this PR

- [ ] Ingestion of categories on annotations (will be covered by #88)
- [ ] Removal of unused JSON archive in snapshot
- [ ] Fix username in exported snapshots (#87)
